### ISSUE TITLE
Savant

### ISSUE THREAD

**Issue by rohrsh** [benumc/bridge-files#1]
Hi buddy. Sorry to hit you up from nowhere!

I've been trying desperately to get a Home Assistant driver working with Savant. I first tried  https://github.com/nonnac/Hass-Savant and it kinda works but not perfectly.  

So I thought I would try your fork to see if it was better. I suspect it needs this bridge to be running also on Savant host? If you recommend this path I do appreciate any tips you have on installing it and making Savant work.

Happy to work on it together too - have some thoughts on how to improve the profile
* Instead of calling Haas api/services/light/turn_on, use the more generic homeassistant/turn_on and then we can drive entities that are not lights. 
* I haven't got brightness status working right yet

Thanks so much for everything you've done so far. You are the first person I've found who also works with both Savant and Home Assistant. 

My terminal output looks like this: 

```
13:08:55.895 Input Message: Control software started successfully.

13:08:56.095 Input Message: Remote Device Manufacturer: Hass

13:08:56.096 Input Message: Remote Device Model: Websocket

13:08:56.097 Input Message: Remote Device Profile Version: hass_websocket.xml (2.0)

13:08:56.097 Input Message: Remote Device Control Port: Ethernet, IP (ip address: x.x.x.x158, port: 25804), XXXX

13:08:56.098 Input Message: Remote Device Controller: Smart Host with Control, 1.1:1

13:08:56.099 Input Message: Remote Device Racepoint OS: 9.4.5:1

13:08:56.099 Input Message: Remote Device Host: Linux Kernel 4.14.78+ga29650fb784a, i.MX6 Quad, Smart Host-10

13:08:58.705 SAVANT Output Message: curl-d '{"entity_id": "light.study_screen_led"}' '/api/services/light/turn_off'

13:09:00.915 SAVANT Output Message: curl-d '{"entity_id": "light.study_screen_led"}' '/api/services/light/turn_on'

13:09:13.241 SAVANT Output Message: \h0A---1-SVC_GEN_GENERIC-RunCLIProgram:COMMAND_STRING=grep -A9999 'env ruby' '$(echo ~/*/*/*/u*/*/hass_websocket.xml)'|ruby\h0A

13:09:16.253 SAVANT Output Message: \h0A---1-SVC_GEN_GENERIC-RunCLIProgram:COMMAND_STRING=grep -A9999 'env ruby' "$(echo ~/*/*/*/*/u*/*/hass_websocket.xml)"|ruby\h0A

13:09:18.274 SAVANT Output Message: curl'/api/states/light.study_screen_led'
```

**Comment by benumc** [comment]
I think the issue you are seeing has been fixed in the version I just pushed. As for actual hass protocol I'm not too sure as I only made the websocket script because of a request from the original profile creator.

You don't need a bridge for this to work.

I fired up a small Hass server but it only reports the default weather.

**Comment by rohrsh** [comment]
Thanks dude. 

I grabbed your latest, but it's still the same experience for me. The messages don't appear to be properly formed (see below RPMTerminal), and there's no responses coming back from my Home Assistant. 

<img width="1049" alt="image" src="https://user-images.githubusercontent.com/38928979/172031976-f5b5e0ea-9700-44b4-b6a2-98030b3a560e.png">


<img width="1050" alt="image" src="https://user-images.githubusercontent.com/38928979/172032046-3a79b742-7d62-4da0-965c-7eeac6524818.png">
 

When I used the driver from the master branch it was exchanging messages - see this comment for an example  - 

https://github.com/nonnac/Hass-Savant/issues/11#issue-1255949825
